src/soter/openssl/soter_rsa_key_pair_gen.c

```diff
@@ -76,9 +76,8 @@ soter_status_t soter_rsa_key_pair_gen_init(soter_rsa_key_pair_gen_t* ctx, const
                                           EVP_PKEY_CTRL_RSA_KEYGEN_BITS,
```

Comment:
Should we also make above return code checking more strict here like in `boringssl`?

Reply:
Do you mean replacing
```c
SOTER_IF_FAIL((1 <= EVP_PKEY_CTX_ctrl(...), ...);
```
with
```c
if (1 > EVP_PKEY_CTX_ctrl(...)) {
    // ...
    return SOTER_FAIL;
}
```
If so then they are equivalent (barring usage of macros).

Reply:
Right, my eyes were confused with different comparison directions. Consistency would be nice though, but can be done separately.